artemis/src/main/java/tech/pegasys/artemis/cli/BeaconNodeCommand.java

```diff
@@ -0,0 +1,499 @@
+      arity = "1")
+  private String network = DEFAULT_NETWORK;
+
+  // P2P
+  @Option(
+      names = {P2P_ENABLED_OPTION_NAME},
+      paramLabel = "<BOOLEAN>",
+      description = "Enables peer to peer",
+      arity = "1")
+  private boolean p2pEnabled = DEFAULT_P2P_ENABLED;
+
+  @Option(
+      names = {P2P_INTERFACE_OPTION_NAME},
+      paramLabel = "<NETWORK>",
+      description = "Peer to peer network interface",
+      arity = "1")
+  private String p2pInterface = DEFAULT_P2P_INTERFACE;
+
+  @Option(
+      names = {P2P_PORT_OPTION_NAME},
+      paramLabel = "<INTEGER>",
+      description = "Peer to peer port",
+      arity = "1")
+  private int p2pPort = DEFAULT_P2P_PORT;
+
+  @Option(
+      names = {P2P_DISCOVERY_ENABLED_OPTION_NAME},
+      paramLabel = "<BOOLEAN>",
+      description = "Enables discv5 discovery",
+      arity = "1")
+  private boolean p2pDiscoveryEnabled = DEFAULT_P2P_DISCOVERY_ENABLED;
+
+  @Option(
+      names = {P2P_DISCOVERY_BOOTNODES_OPTION_NAME},
+      paramLabel = "<enode://id@host:port>",
+      description = "ENR of the bootnode",
+      split = ",",
+      arity = "0..*")
+  private ArrayList<String> p2pDiscoveryBootnodes = DEFAULT_P2P_DISCOVERY_BOOTNODES;
+
+  @Option(
+      names = {P2P_ADVERTISED_IP_OPTION_NAME},
+      paramLabel = "<NETWORK>",
+      description = "Peer to peer advertised ip",
+      arity = "1")
+  private String p2pAdvertisedIp = DEFAULT_P2P_ADVERTISED_IP;
+
+  @Option(
+      names = {P2P_ADVERTISED_PORT_OPTION_NAME},
+      paramLabel = "<INTEGER>",
+      description = "Peer to peer advertised port",
+      arity = "1")
+  private int p2pAdvertisedPort = DEFAULT_P2P_ADVERTISED_PORT;
+
+  @Option(
+      names = {P2P_PRIVATE_KEY_FILE_OPTION_NAME},
+      paramLabel = "<FILENAME>",
+      description = "This node's private key file",
+      arity = "1")
+  private String p2pPrivateKeyFile = DEFAULT_P2P_PRIVATE_KEY_FILE;
+
+  @Option(
+      names = {P2P_PEER_LOWER_BOUND_OPTION_NAME},
+      paramLabel = "<INTEGER>",
+      description = "Lower bound on the target number of peers",
+      arity = "1")
+  private int p2pLowerBound = DEFAULT_P2P_PEER_LOWER_BOUND;
+
+  @Option(
+      names = {P2P_PEER_UPPER_BOUND_OPTION_NAME},
+      paramLabel = "<INTEGER>",
+      description = "Upper bound on the target number of peers",
+      arity = "1")
+  private int p2pUpperBound = DEFAULT_P2P_PEER_UPPER_BOUND;
+
+  @Option(
+      names = {P2P_STATIC_PEERS_OPTION_NAME},
+      paramLabel = "<PEER_ADDRESSES>",
+      description = "Static peers",
+      arity = "1")
+  private ArrayList<String> p2pStaticPeers = DEFAULT_P2P_STATIC_PEERS;
+
+  // Interop
+
+  @Option(
+      names = {X_INTEROP_GENESIS_TIME_OPTION_NAME},
+      paramLabel = "<INTEGER>",
+      description = "Time of mocked genesis",
+      arity = "1")
+  private Integer xInteropGenesisTime = DEFAULT_X_INTEROP_GENESIS_TIME;
+
+  @Option(
+      names = {X_INTEROP_OWNED_VALIDATOR_START_INDEX_OPTION_NAME},
```

Comment:
Are these '--x' prefixed options meant to be hidden?

Reply:
done